samples/alter-headers/README.md

```diff
@@ -0,0 +1,13 @@
+# Altering response headers
+
+This sample shows how to alter the headers as you stream a response from an origin
+server back to a client.
+
+You can set the origin site via the `origin_host` variable, either by editing `spin.toml`
+or by overriding it on the command line:
+
+```
+SPIN_VARIABLE_ORIGIN_HOST=example.com spin up --build
+```
+
+(Note this should _not_ include the `https://` prefix - just the host name.)
```

Comment:
You could add a curl command here to view the headers

Something similar to this or a `curl -v` might be more well known, idk
```
$ curl -I http://localhost:3000
HTTP/1.1 200 OK
alt-svc: h3=":443"; ma=93600,h3-29=":443"; ma=93600,quic=":443"; ma=93600; v="43"
cache-control: max-age=941
date: the eleventy-sixth of June
etag: "84238dfc8092e5d9c0dac8ef93371a07:1736799080.121134"
last-modified: Mon, 13 Jan 2025 20:11:20 GMT
friendly-message: Hello from FWF
```

Reply:
@adamreese Good call (especially important since I masterfully defaulted to an origin site that needs a route).  I found `curl -i` (lowercase) which shows the altered headers and the body being passed through unchanged.